packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -190,7 +190,7 @@ class AndroidGradleBuilder implements AndroidBuilder {
     @required FlutterProject project,
     @required Set<AndroidBuildInfo> androidBuildInfo,
     @required String target,
-    @required String outputDirectoryPath,
+    String outputDirectoryPath,
```

Comment:
Null `outputDirectoryPath` defaults to `project.android.buildDirectory` on line 197.